hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -806,10 +790,14 @@ private static void logFSTree(Logger LOG, final FileSystem fs, final Path root,
 
     for (FileStatus file : files) {
       if (file.isDirectory()) {
-        LOG.debug(prefix + file.getPath().getName() + "/");
+        if (LOG.isDebugEnabled()) {
```

Comment:
Does all of logFSTree only log at debug? Can we just do this guard for the whole method contents and skip the traversal?

Reply:
Great catch.  I'll change.  Also, I noticed that LOG here is being passed into the method as an argument.  This is dangerous as it matches the class's static instance of `LOG`.

Reply:
it's a private method, do we actually need to pass in a LOG object?

Reply:
Ya, the Logger gets ultimately passed in from other classes.  I've made the requested change.